ontopy/excelparser.py

```diff
@@ -9,18 +9,23 @@
 
 Note that correct case is mandatory.
 """
-import sys
+from typing import Tuple, Union, Sequence
 import warnings
-from typing import Tuple, Union
-import pyparsing
+
 import pandas as pd
+import pyparsing
+
 import ontopy
```

Comment:
Is there a specific reason for the change in order of the imported packages?

Reply:
Yes there are conventions for this :-) By convention all standard libraries are imported first, followed by an empty line. Then you import other third party packages followed by yet an empty line. Finally you do the imports from the current package (which could be relative imports). Within each category I usually order them with the most fundamental packages first (according to my subjective judgement) while Casper argue that they should be ordered alphabetically.

Reply:
Thank you :)